Source/URedis/Private/URedis.cpp

```diff
@@ -44,13 +43,22 @@ bool FURedis::Set(FStringView key, FStringView value) const {
 }
 
 TOptional<FString> FURedis::Get(FStringView key) const {
-    auto result = _instance->get(TCHAR_TO_UTF8(key.GetData()));
+    auto result{_instance->get(TCHAR_TO_UTF8(key.GetData()))};
```

Comment:
What is the difference?

Reply:
This is brace initialization and I recommend preferring this initialization type over the `=`. There are several advantages with [numbers](https://stackoverflow.com/a/18222927/2891324) and with [strings as well](https://stackoverflow.com/a/9977266/2891324). In short, in general, this initialization type has to be preferred